client/src/Movies/SavedList.js

```diff
@@ -10,9 +10,9 @@ export default class SavedList extends Component {
       <div className="saved-list">
         <h3>Saved Movies:</h3>
         {this.props.list.map(movie => (
-          <span className="saved-movie">{movie.title}</span>
+          <NavLink to={`/movies/${movie.id}`} key={movie.id}><span className="saved-movie">{movie.title}</span></NavLink>
```

Comment:
This line would be more readable if you split it up:
```javascript
<NavLink to={`/movies/${movie.id}`} key={movie.id}>
  <span className="saved-movie">{movie.title}</span>
</NavLink>
```
or, better:
```javascript
<NavLink to={`/movies/${movie.id}`} key={movie.id}>
  <span className="saved-movie">
    {movie.title}
  </span>
</NavLink>
```
or _maybe_ even better:
```javascript
<NavLink 
  to={`/movies/${movie.id}`} 
  key={movie.id}
>
  <span className="saved-movie">
    {movie.title}
  </span>
</NavLink>
```